<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupConnection</class>
 <widget class="QDialog" name="SetupConnection">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройка соединения</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="HostName_Label">
     <property name="text">
      <string>Хост:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="HostName_Edit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="DBName_Label">
     <property name="text">
      <string>База данных:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="DBName_Edit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="UserName_Label">
     <property name="text">
      <string>Имя пользователя:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="UserName_Edit"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="Password_Label">
     <property name="text">
      <string>Пароль:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="Password_Edit">
     <property name="inputMask">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="TryConnect_Button">
     <property name="text">
      <string>Подключиться</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="port_Label">
     <property name="text">
      <string>Порт:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="Port_Edit">
     <property name="text">
      <string>5432</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
